src/test/java/de/rabea/game/GameSetupTest.java

```diff
@@ -32,6 +33,16 @@ public void setsUpFirstHumanVsHuman3x3Game() {
     }
 
     @Test
+    public void playsHuman3x3GameTwice() {
+        fakeUserInterface.provideConsoleInput("1", "3", "1", "7", "3", "4", "2", "y", "1", "3", "2", "5", "9", "7", "3", "6", "4", "8", "1", "n");
```

Comment:
Maybe not an issue right now, but a test double like that leaks concrete details of one user interface into your tests (at the moment the console).
Can you think of a way where the UI double could be used and set up in a way that makes it agnostic to a UI that's used in production?
(How - if at all - would this code need to change if you think of your upcoming JavaFX GUI?)